examples/java_springboot/Makefile

```diff
@@ -1,47 +1,38 @@
-.PHONY: run watch test-unit test-functional test format style complexity
+.PHONY: run watch test-unit test-functional test-integration test format style complexityall prepare
```

Comment:
missing space between `complexity` and `all`

Reply:
good catch !